db/queries.py

```diff
@@ -3,44 +3,60 @@
 
 
 def get_messages_that_contain_word(word: str) -> list[Message]:
-    pass
+    return list(Message.objects.filter(text__icontains=word))
 
 
 def get_untitled_chats() -> list[Chat]:
-    pass
+    return list(Chat.objects.filter(title__startswith="Untitled"))
 
 
 def get_users_who_sent_messages_in_2015() -> list[str]:
-    pass
+    return list(Message.objects.filter(sent__year=2015).
```

Comment:
You need to select Users, not Message. Start your query with User.objects.

Reply:
Also format better this)
```suggestion
    return list(
        Message.objects.filter(sent__year=2015).
```